File: services/internal-treasury/src/main/java/br/com/deltaglobalbank/internal_treasury/infrastructure/web/request/PaysmartPixRequest.java
package br.com.deltaglobalbank.internal_treasury.infrastructure.web.request;

import java.math.BigDecimal;

public record PaysmartPixRequest(
    long accountId,
    String recipientInstitutionCode,
    String recipientBranchCode,
    String recipientAccountNumber,
    String recipientAccountType,
    String recipientName,
    BigDecimal operationAmount
) {
}

File: services/internal-treasury/src/main/kotlin/br/com/deltaglobalbank/internal_treasury/infrastructure/web/request/PaysmartPixRequest.kt
package br.com.deltaglobalbank.internal_treasury.infrastructure.web.request

import java.math.BigDecimal

data class PaysmartPixRequest (
    val accountId: Long,
    val recipientInstitutionCode: String,
    val recipientBranchCode: String,
    val recipientAccountNumber: String,
    val recipientAccountType: String,
    val recipientName: String,
    val operationAmount: BigDecimal
)